tests/sentry/grouping/test_normalize_message.py

```diff
@@ -122,12 +122,12 @@ def record_analytics(monkeypatch):
         (
             "Uniq ID - sql savepoint",
             '''SQL: RELEASE SAVEPOINT "s140177518376768_x2"''',
-            '''SQL: RELEASE SAVEPOINT "<uniq_id>"''',
+            """SQL: RELEASE SAVEPOINT <uniq_id>""",
```

Comment:
I believe this experiment will work very well.

Because we're using project IDs to control this experiment, all these tests are being updated. In general, we need to override the option for the tests to work.

What's our plan to roll this out? This could surprise a lot of customers who are expecting an issue per customer ID.

We need to discuss it before we go all out.